config/scripts/setup-test-data.sh

```
@@ -5,25 +5,29 @@ set -e
 
 echo "🏗️ Setting up WordPress and SportsPress test data..."
 
+# Skip setup if already completed (idempotency guard)
+if [ -f /var/lib/baseline/baseline.sql ]; then
+    echo "✅ Setup already completed (baseline exists). Skipping."
+    exit 0
+fi
+
 DB_HOST=${WORDPRESS_DB_HOST:-localhost}
 DB_USER=${WORDPRESS_DB_USER:-wordpress}
```

Shellcheck (0.11.0)

[warning] DB_USER appears unused. Verify use (or export if used externally).

(SC2034)